### DISCUSSION TITLE
No Traffic to bond0

### DISCUSSION THREAD

**Discussion by wubbaroo** [Security-Onion-Solutions/securityonion#4075] [Unsupported Versions]
Hello, I have Ubuntu 18.04 running Security Onion 2. There is no traffic getting to bond0 interface, however there is traffic to eno4. I've ran the following command:

_so-monitor-add eno4_

However, this no traffic is following to bond0. I've tried manually forwarding the traffic by enabling forwarding in /etc/sysctl.conf and adding the following iptables rules:

_sudo iptables -t nat -A POSTROUTING --out-interface bond0 -j MASQUERADE  
sudo iptables -A FORWARD --in-interface eno4 -j ACCEPT_

Any thoughts?

**Comment by weslambert** [comment]
What is the output of the following?

`nmcli con show`
